build-cmd.sh

```diff
@@ -93,32 +93,41 @@ pushd "$ZLIB_SOURCE_DIR"
             # We copy libz.a for package, not dylibs
             install_name="@executable_path/../Resources/libz.1.dylib"
 
-            cc_opts="${TARGET_OPTS:--arch $AUTOBUILD_CONFIGURE_ARCH $LL_BUILD_RELEASE}"
-            ld_opts="-Wl,-install_name,\"${install_name}\" -Wl,-headerpad_max_install_names"
-            export CC=clang
-
-            # release
-            CFLAGS="$cc_opts" \
-            LDFLAGS="$ld_opts" \
-                ./configure $cfg_sw --prefix="$stage" --includedir="$stage/include/zlib-ng" --libdir="$stage/lib/release" --zlib-compat
-            make
-            make install
-
-            # conditionally run unit tests
-            if [ "${DISABLE_UNIT_TESTS:-0}" = "0" ]; then
-                # Build a Resources directory as a peer to the test executable directory
-                # and fill it with symlinks to the dylibs.  This replicates the target
-                # environment of the viewer.
-                mkdir -p ../Resources
-                ln -sf "${stage}"/lib/release/*.dylib ../Resources
-
-                make test
-
-                # And wipe it
-                rm -rf ../Resources
-            fi
-
-            make distclean
+            export MAKEFLAGS="-j${AUTOBUILD_CPU_COUNT:-2}"
+
+            # todo - build support into autobuild for universal AUTOBUILD_CONFIGURE_ARCH
```

Comment:
What would that involve?

Reply:
so, unfortunately this probably isn't possible the way I had envisioned.  apple's toolchain now lets you transparently build fat objects/archives/dylibs/executables simply by passing multiple arch flags in the appropriate CFLAGS places etc.  unfortunately almost every real project has some logic in its build system for selecting appropriate architecture specific source files and preprocessor defines, which conflict with this method.

anyways, if the new transparent method had worked, I would have wanted to build support for it into autobuild, but seems like that won't be useful